source/manual/connect-to-vcloud-director.html.md

```diff
@@ -4,39 +4,55 @@ title: Connect to vCloud Director (Carrenza only)
 section: Infrastructure
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-04-24
+last_reviewed_on: 2019-08-08
 review_in: 6 months
 ---
 
 vCloud Director is the interface we use to manage our infrastructure in Carrenza.
 This includes virtual machines, gateways, firewalls and VPNs between providers.
 
 To access vCloud Director, you will need to connect to a Carrenza-provided VPN.
-You can use either Cisco AnyConnect or openconnect to do this.
+You can use either Cisco AnyConnect or OpenConnect as a VPN client for this.
 
-## Prerequisites
+## Setting up the Cisco AnyConnect VPN profile on a Mac
 
-1. Get the VPN client certificate from the [2nd line password store](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/carrenza/vpn-certificate.gpg).
-2. Save the certificate to a file on your machine (eg. `vcloud.pem`).
-3. Get the VPN credentials, also from the 2nd line password store (e.g.
-for integration this would be in [vcloud-integration.gpg](https://github.com/alphagov/govuk-secrets/tree/master/pass/2ndline/carrenza) ). You can find
-additional instructions about decrypting these files [here](https://github.com/alphagov/govuk-secrets/blob/master/pass/README.md) under the heading "Get a password".
-4. You'll need to use an app (such as Google Authenticator) to turn the TOTP
-   secret into a 2FA code.
-
-## Connecting with Cisco AnyConnect
-
-1. Convert the VPN client certificate from PEM format to PFX format by running
-   `openssl pkcs12 -export -in vcloud.pem -out vcloud.pfx`. You'll be asked for
-   two passwords. For the first one, enter the VPN password, and for the second
-   one, enter the certificate passphrase.
-2. Import the PFX format certificate into your Keychain by running
-   `security import vcloud.pfx -k ~/Library/Keychains/login.keychain-db`.
-   You'll be asked for a password. Enter the certificate passphrase.
-3. Create a new file on your machine at `/opt/cisco/anyconnect/profile/carrenza-secure.xml`.
+1. Clone the govuk-secrets private Git repo. The repo is very large but there is no need to download the full history.
+```
+mkdir -p ~/govuk && cd ~/govuk
+git clone --depth 10 git@github.com:alphagov/govuk-secrets.git
+```
+2. Get the VPN client certificate and private key from the 2nd line [password store](https://github.com/alphagov/govuk-secrets/tree/master/pass) and save the decrypted contents to a file on your machine (for example ~/carrenza-vpn-cert-and-key.pem).
+```
+cd govuk-secrets/pass
+export PASSWORD_STORE_DIR=2ndline
+pass carrenza/vpn-certificate >~/carrenza-vpn-cert-and-key.pem
```

Comment:
Space needed here like this `> ~/carrenza-vpn-cert-and-key.pem` ?

Reply:
> Space needed here like this `> ~/carrenza-vpn-cert-and-key.pem` ?

It's not syntactically needed but happy to add it if you think it helps readability.

Reply:
Done :)